### DISCUSSION TITLE
GUI on remote daemon host can't be loaded after changing self_hostname

### DISCUSSION THREAD

**Discussion by sang-ook** [Chia-Network/chia-blockchain#8125] [Support]
I'm trying to setup a remote daemon host on Windows machine to which I want to connect to from Linux machine according to guide referenced [here](https://github.com/Chia-Network/chia-blockchain/wiki/Connecting-the-UI-to-a-remote-daemon).

On the daemon host I've changed line **self_hostname** from _localhost_ to _0.0.0.0_. After that I've started Chia GUI app on daemon host. Unfortunately app doesn't go past the loading screen and when I've checked logs, I've found this message:

`: WARNING  Cannot connect to daemon at ws://0.0.0.0:55400`

Any help is appreciated.

**Comment by BrandtH22** [comment]
@sang-ook , the newest version(s) of chia has fixes in place to resolve this issue. Please download it from here: https://www.chia.net/downloads/

Since this issue has been resolved with the latest version(s) of chia we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
